umbraco/Umbraco-CMS
# V8: Email Marketing Opt In

## Test Notes
* Review code of PR and no console.logs left laying about etc
* Ensure you try with a clean/brand new install (be it from build powershell Umbraco zip or similar)
* Login to backoffice
* Original intro tour will display
* New Email Marketing Tour **ONLY displays if intro tour is disabled OR has been completed fully**
* Accept Yes and it will do a HTTP POST to the Azure function which in turn adds you to the email marketing service being used currently. Ask Lars in COMA to confirm that your email and associated usergroups ended up in the system
* Reset your test - update the JSON data stored in `UmbracoUsers` DB table in the column `TourData`
* Log out of Umbraco & back in again to see tour again
* Answer NO or click the X to dismiss, the tour will be marked as disabled in the JSON stored in the DB and will no longer appear, after dismissing it the intro tour should display again but only if you did not dismiss it or reset the data in the JSON in the DB table.

## Any Questions
Reach out & let me know
